src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -234,13 +234,15 @@ extern "C" {
 
 /**
  * Max update version length in chars
+ * https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_DeleteSignalingChannel.html#KinesisVideo-DeleteSignalingChannel-request-CurrentVersion
  */
-#define MAX_UPDATE_VERSION_LEN                   128
+#define MAX_UPDATE_VERSION_LEN                   64
 
 /**
  * Max ARN len in chars
+ * https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_DescribeSignalingChannel.html#API_DescribeSignalingChannel_RequestSyntax
  */
-#define MAX_ARN_LEN                              2048
+#define MAX_ARN_LEN                              1024

```

Comment:
Oh! I just realized that we can't change this as the ARN can be 2048 for other resources such as KMS. We use the same ARN len macro for tags, KMS and KVS. Some are 2048.

https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_CreateStream.html#KinesisVideo-CreateStream-request-KmsKeyId

Please revert this one

Reply:
Ouch. Indeed, reverted this line. thanks for your review. :)